charts/hivemq-platform/templates/hivemq-custom-resource.yml

```diff
@@ -186,14 +186,18 @@ spec:
               {{- include "hivemq-platform.get-additional-volumes" . | indent 12 }}
             {{- end }}
           {{- end }}
-          {{- if and (typeIsLike "map[string]interface {}" .Values.podScheduling.affinity) (.Values.podScheduling.affinity) }}
+          {{- with .Values.podScheduling.affinity }}
```

Comment:
Just for reference: https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with

Reply:
This is really nice, much easier to read (once you know how `with` works)!